### PR TITLE
UI general refactor

### PR DESCRIPTION
**Pull request by YMari**

### CODE REVIEW

File: premarum-web-client/utility/helpers/selectionToRow.ts [hunk 1]
```diff
@@ -17,9 +17,24 @@ export async function GetRows(selections: IPreEnrollmentSelectionResponse[]) {
...
+        times = [...new Set(times)]; // remove duplicates
+
+        result.push({
+            id: parseInt(i), 
+            entryId: selections[i].id, 
+            course: selections[i].course.courseCode, 
+            section: selections[i].sectionName,
+            credits: selections[i].course.courseCredit, 
+            classroom: selections[i].classRoom,
+            timeslot: days.join(", ") + "\n" + times.join(", "),
```
Comment (kenneth-rosario) [inline premarum-web-client/utility/helpers/selectionToRow.ts:33]: I was thinking this format would be good: "Monday 10:30-11:30, Tuesday 11:50-12:50" Or we can also abbreviate to Mon Tue, ...

File: premarum-web-client/components/catalogGrid.tsx [hunk 2]
```diff
@@ -43,6 +44,12 @@ type AddSelectionProps = {
 export function AddSelectionButton({preEnrollmentId, selectionsRef, changeTab}: AddSelectionProps) {
     const { addSelectionFn } = usePreEnrollment(preEnrollmentId)
     const [isLoading, setIsLoading] = useState(false)
+    const [selected, setSelected] = useState(false)
+
+    useEffect(() => {
```
Comment (kenneth-rosario) [inline premarum-web-client/components/catalogGrid.tsx:49]: Did this work for you?

File: premarum-web-client/components/GithubIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { Button, Tooltip } from "@mui/material";
+
+export default function GithubIcon() {
```
Comment (kenneth-rosario) [inline premarum-web-client/components/GithubIcon.tsx:3]: Cool!

**Review by kenneth-rosario — COMMENTED** [review]
Minor comments

File: premarum-web-client/components/GithubIcon.tsx (hunk 3 above)
Comment (joseriveramorales) [reply premarum-web-client/components/GithubIcon.tsx:3]: 👀 si es lo que pienso que es esta duro

File: premarum-web-client/components/catalogGrid.tsx (hunk 2 above)
Comment (YMari) [reply premarum-web-client/components/catalogGrid.tsx:49]: no

**Review by kenneth-rosario — APPROVED** [review]

**Pull request merged**